ebpfsvc/rpc_util.cpp

```diff
@@ -7,22 +7,23 @@
 
 #pragma comment(lib, "Rpcrt4.lib")
 
-#define RPC_SERVER_ENDPOINT L"\\pipe\\ebpf_service"
+#define RPC_SERVER_ENDPOINT L"ebpfsvc rpc server"
```

Comment:
Don't use hard-coded endpoints. Use dynamic endpoints to avoid name squatting attacks.

Reply:
Fixed.